cmd/runmqserver/webserver.go

```diff
@@ -33,7 +35,22 @@ func startWebServer() error {
 		return nil
 	}
 	log.Println("Starting web server")
-	out, rc, err := command.RunAsMQM("strmqweb")
+	cmd := exec.Command("strmqweb")
+	// Set a default app password for the web server, if one isn't already set
+	_, set := os.LookupEnv("MQ_APP_PASSWORD")
+	log.Println(cmd.Env)
+	if !set {
+		// Take all current environment variables, and add the app password
+		cmd.Env = append(os.Environ(), "MQ_APP_PASSWORD=passw0rd")
+	}
+	log.Println(cmd.Env)
```

Comment:
Same here.

Reply:
That's debug information which I forgot to take out.  Well spotted... the system works!